### DISCUSSION TITLE
Bintray has shut down: where to migrate

### DISCUSSION THREAD

**Discussion by michaelklishin** [rabbitmq/rabbitmq-server#3020] [Other]
Bintray, one of the services our team currently uses to distribute packages, has shut down on May 1st, 2021.

This means the all package downloads from Bintray will be blocked, and no new releases will be possible.

This [blog post](https://blog.rabbitmq.com/posts/2021/03/migrate-off-of-bintray/) describes your migration options
for various RabbitMQ package types.

**Comment by artem-sidorenko** [comment]
Impossible to mirror the new repositories on EL 7/8 :-( https://groups.google.com/g/rabbitmq-users/c/yPl0whVvOXo/m/u10Z-vgeBQAJ

**Reply by michaelklishin** [reply]
As explained in the thread, we do not control PackageCloud repository layout. I could successfully mirror a repository from Cloudsmith.io.

Every RabbitMQ release includes MPLv2-licensed `.rpm` files that we then upload to PackageCloud and Cloudsmith. You
can do the same with a local Yum repo. I'm afraid our team does not have the resources to make repository mirroring a priority,  nor do we have any knowledge about how PackageCloud works.

This is open source software so you can build packages of your own or upload the provided ones to any Yum repository.
